tests/components/totalconnect/common.py

```diff
@@ -11,7 +11,7 @@
 
 from tests.common import MockConfigEntry
 
-LOCATION_ID = "123456"
+LOCATION_ID = 123456
```

Comment:
I think this should be an integer, based on:
https://github.com/home-assistant/core/blob/71d35a03e17b4b48c1c33df541377b92b6cfd3b1/homeassistant/components/totalconnect/alarm_control_panel.py#L97

However, it would be best to have confirmation from a read user.